packages/lib/src/index.js

```diff
@@ -18,13 +17,14 @@ const assertions = helpers.assertions
 const assertRevert = helpers.assertRevert
 
 // model objects
+import Proxy from './proxy/Proxy'
```

Comment:
I think we don't need to import this one right?

Reply:
It's exported in L40 of this file